Delete user

charurajput16 opened the pull request:
## Describe your changes
Added functionality to delete the user from backend

## Issue number

## Please ensure all items are checked off before requesting a review:

- [x] I deployed the code locally.
- [x] I have performed a self-review of my code.
- [ ] I have included the issue # in the PR.
- [x] I have labelled the PR correctly.
- [ ] The issue I am working on is assigned to me.
- [x] I didn't use any hardcoded values (otherwise it will not scale, and will make it difficult to maintain consistency across the application).
- [x] I made sure font sizes, color choices etc are all referenced from the theme.
- [x] My PR is granular and targeted to one specific feature.
- [x] I took a screenshot or a video and attached to this PR if there is a UI change.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **New Features**
	- Enhanced team member deletion now communicates with the server to provide immediate success or error notifications when you delete a member, ensuring a more reliable and informative experience.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

MuhammadKhalilzadeh reviewed (changes requested):
Hi @charurajput16 
It's not working

https://github.com/user-attachments/assets/d3d3484b-1d6d-498f-9022-3bbdba056929

charurajput16 commented:
@MuhammadKhalilzadeh - It is based on the changes done by the @HarshP4585 . He updated and fixed the API. As soon as we have updated code on Docker, it will work.
